src/main/kotlin/no/nav/syfo/client/pdl/PdlPersonResponse.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.syfo.client.pdl
+
+import java.io.Serializable
+
+data class PdlPersonResponse(
+    val errors: List<PdlError>?,
+    val data: PdlHentPerson?,
+)
+
+data class PdlHentPerson(
+    val hentPerson: PdlPerson?,
+) : Serializable
+
+data class PdlPerson(
+    val adressebeskyttelse: List<Adressebeskyttelse>?,
+) : Serializable
+
+data class Adressebeskyttelse(
+    val gradering: Gradering,
+) : Serializable
+
+enum class Gradering : Serializable {
+    STRENGT_FORTROLIG_UTLAND,
+    STRENGT_FORTROLIG,
+    FORTROLIG,
+    UGRADERT,
+}
+
+fun PdlHentPerson.isKode6(): Boolean {
```

Comment:
Jeg er ikke sikker på om dette er riktig plass for disse 🤔 Burde det kanskje ligge i en service, eller en util? 🤷